main/.vitepress/config.mjs

```diff
@@ -510,5 +513,30 @@ export default defineConfig({
         },
       },
     },
+    footer: {
+      copyright: `© ${new Date().getFullYear()} Agoric Systems Operating Company. All Rights Reserved.`,
```

Comment:
is this copyright year computed at build time? or does it rely on the client to compute it?

Reply:
I believe this happens at build time